docs/software/uenv/management.md

```diff
@@ -0,0 +1,364 @@
+    eccodes/cug:1800983313           gh200  daint   cab8e857967d0708   4,036    2025-05-05
+    editors/24.7:v2                  gh200  daint   e7b0d930df729da5   1,270    2024-09-04
+    gromacs/2024:v1                  gh200  daint   b58e6406810279d5   3,658    2024-09-12
+    ```
+
+[](){#ref-uenv-image-add}
+### `uenv image add`
+
+To add a uenv SquashFS file to a repo, so that it can be used with a [label][ref-uenv-labels], use the `uenv image add` command.
+
+!!! example "add an image to the default repo"
+    ```console
+    $ uenv image add myenv/24.7:v1@eiger%zen2 ./store.squashfs
+    ```
+
+    Note that the [label][ref-uenv-labels] must be of the complete `name/version:tag@system%uarch` form.
+
+[](){#ref-uenv-repo}
+## Repositories
+
+A repository is a directory that contains an sqlite database `index.db` in the root, and an `images` sub-directory that contains the individual uenv that have been downloaded in directories.
+
+```
+repo
+├── index.db
+└── images
+    ├── 01ed...a5ab
+    │   └── store.squashfs
+    ├── 0b6a...24c1
+    │   └── store.squashfs
+    └── 1528...785a
+        └── store.squashfs
+```
+
+### Creating and using repositories
+
+A repo will automatically be created in your [Scratch path][ref-storage-scratch] attached to the cluster you are on when you first use uenv.
+This _default repo_ is used by all calls to uenv, unless it is overridden using the options in this section.
+
+!!! question "Where is my repo?"
+    The Scratch filystem used depends on the cluster:
+
+    | cluster | repo path | notes |
+    | ------- | --------- | ----- |
+    | Eiger, Daint | `/capstor/scratch/cscs/$USER/.uenv-images` | will move to `/ritom/scratch/cscs/$USER/.uenv-images` when it replaces capstor in December 2025|
+    | Santis       | `/capstor/scratch/cscs/$USER/.uenv-images` |       |
+    | Clariden     | `/iopsstor/scratch/cscs/$USER/.uenv-images` |       |
+    | Others       | `$SCRATCH/.uenv-images` | `$HOME/.uenv/repo` on systems with no `SCRATCH` defined. |
+
+The location of your default repo can be changed by setting the `repo` field in the [uenv config file][ref-uenv-configure-options-repo].
+For example, the following would use a shared repo in a project's [Store][ref-storage-store]:
+```
+repo = /capstor/store/cscs/userlab/sm42/team-uenv
+```
+
+The `--repo` flag can also be used to use a specific repo on the command line, for example to list all uenv in the repo ``/capstor/store/cscs/userlab/sm42/team-uenv`:
+
+```console
+$ uenv --repo=/capstor/store/cscs/userlab/sm42/team-uenv image ls
+```
+Note that the `--repo` flag goes between `uenv` and the command, `image ls` in the example above, and can be used with all uenv commands.
+
+[](){#ref-uenv-repo-status}
+### `uenv repo status`
+
+The `uenv repo status` command provides information about a repository.
+By default, `uenv repo status` will print information about the default repo.
+To get the status of a different repo, provide it as an optional argument:
```

Comment:
side-comment: here you mean it as "positional optional argument" right? not as in `--repo` flag. does also the flag work?

Reply:
Yes. I have made it clearer.
The `--repo` flag will also work.